packages/webhooks/model-validator/validator.js

```diff
@@ -0,0 +1,208 @@
+
+            if (trimmedString[i] === '{' || trimmedString[i] === '}') {
+                expandedChars.push(' ', trimmedString[i], ' ')
+            } else {
+                expandedChars.push(trimmedString[i])
+            }
+        }
+
+        return expandedChars.join('').trim().split(/\s+/).join(' ')
+    }
+
+    /**
+     * Accepts generated JSON-schema and moves every property types to boolean
+     * Also flattens arrays types and adds minimal amount of specified properties to schema
+     * @param {any} schema
+     * @returns {any}
+     * @private
+     */
+    static _maskSchemaTypes (schema) {
+        if (typeof schema !== 'object') {
+            return schema
+        }
+
+        // NOTE: Deep cloning JSON object
+        const newSchema = JSON.parse(JSON.stringify(schema))
+
+        const type = get(newSchema, 'type')
+        if (type) {
+            if (type === 'object') {
+                for (const key of Object.keys(newSchema.properties)) {
+                    newSchema.properties[key] = WebHookModelValidator._maskSchemaTypes(newSchema.properties[key])
+                    newSchema.minProperties = 1
+                }
+
+                return newSchema
+            } else if (type === 'array') {
+                return WebHookModelValidator._maskSchemaTypes(newSchema.items)
+            } else {
+                return {
+                    type: 'boolean',
+                }
+            }
+        } else {
+            for (const key of Object.keys(newSchema)) {
+                newSchema[key] = WebHookModelValidator._maskSchemaTypes(newSchema[key])
+            }
+
+            return newSchema
+        }
+    }
+
+    /**
+     * Tries to build a JSON sample from normalized field-string to pass it into validator
+     * created by _maskSchemaTypes
+     * @param {string} fieldsString
+     * @returns {JSON}
+     * @private
+     */
+    static _buildSampleFromFieldsString (fieldsString) {
+        const parts =  fieldsString.split(' ')
+        const mappedParts = parts.map((part, index) => {
+            if (part === '{') {
+                return part
+            } else {
+                let result
+                if (part === '}' || part === '{') {
+                    result = part
+                } else if (index < parts.length - 1) {
+                    result = parts[index + 1] === '{' ? `"${part}":` : `"${part}": true`
+                } else {
+                    result = `"${part}": true`
+                }
+
+                if (index < parts.length - 1 && parts[index + 1] !== '}' && parts[index + 1] !== '{') {
+                    result += ','
+                }
+
+                return result
+            }
+        })
+
+        try {
+            const sample = JSON.parse(mappedParts.join(' '))
+            return { data: sample, error: null }
+        } catch (error) {
+            return { data: null, error: 'Cannot convert fields string to JSON sample object.' }
```

Comment:
The error message may be helpful.
```suggestion
            return { data: null, error: `Cannot convert fields string to JSON sample object: ${error.message}` }
```